frame/elections-phragmen/src/migrations/v5.rs

```diff
@@ -0,0 +1,65 @@
+use super::super::*;
+//  use frame_support::traits::Get;
+
+/// Migrate the locks and vote stake on accounts that have more than their free balance locked.
+///
+/// This migration addresses a bug were a voter could lock up to their reserved balance + free
+/// balance. Since locks are only designed to operate on free balance, this put those affected in a
+/// situation where they could increase their free balance but still not be able to use there funds
+/// because it is less than the lock.
+pub fn migrate<T: Config>() -> Weight {
+	let mut weight = 0;
+
+	for (who, mut voter) in Voting::<T>::iter() {
```

Comment:
could also use `::translate`.

Reply:
From my understanding translate would be overkill here since it would re-write every map entry. Additionally it seems non-idiomatic since it looks like it targets use cases where the stored type is being changed

Reply:
I think for the purposes of fixing this on our networks, it would be better to write a custom list of affected accounts, and directly address those versus iterating through all possible voters, as that kind of iteration migration is not really great for parachains or other runtimes where there is limited execution time.

Reply:
and for that, we should wait until the original patch is live, then we will know a final list of affected users.